package.json

```diff
@@ -184,6 +184,7 @@
     "nock": "8.0.0",
     "nodemon": "1.4.1",
     "react-addons-test-utils": "15.4.0",
+    "react-codemod": "github:reactjs/react-codemod#d6563d9",
```

Comment:
There doesn't seem to be an `npm` AFAICS. FWIW, [I added the devDep without SHA stamp](https://github.com/Automattic/wp-calypso/blob/121c5b4c5c459f4e40b491213fa385f546ccae3b/package.json#L191) (thus tracking `master`) a while back.